invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -1,111 +1,170 @@
+          loanState={data.metadata.state}
+          changeItemClickHandler={this.showAvailableItems}
+        />
+      )
+    );
+  }
+
+  _renderDeletedItem() {
+    const data = this.props.loanDetails;
+    return (
+      this._isItemDeleted() && (
+        <Message
+          warning
+          icon="trash alternate"
+          header={`The Item ${
+            data.metadata.item_pid
+          } assigned to this loan has been deleted!`}
+          content={[
+            'If you need further assistance contact ',
+            <a
+              href={`mailto:${
+                invenioConfig.support_email
+              }?subject=Deleted Item ${data.metadata.item_pid}`}
+              key="support-email"
+            >
+              <Icon name="mail" />
+              {invenioConfig.support_email}
+            </a>,
+          ]}
+        />
+      )
+    );
+  }
+
+  _renderLoan() {
+    const data = this.props.loanDetails;
+    return (
+      <Grid padded columns={2}>
+        <Grid.Column width={16}>
+          <Header as="h1">Loan - {data.loan_pid}</Header>
+        </Grid.Column>
 
-          <Grid.Column>
-            <Table basic="very">
-              <Table.Body>
-                <Table.Row>
-                  <Table.Cell width={4}>Document pid</Table.Cell>
-                  <Table.Cell width={12}>
-                    {data.metadata.document_pid}
-                  </Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>Item pid</Table.Cell>
-                  <Table.Cell width={12}>{data.metadata.item_pid}</Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>Patron pid</Table.Cell>
-                  <Table.Cell width={12}>{data.metadata.patron_pid}</Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>Pickup Location pid</Table.Cell>
-                  <Table.Cell width={12}>
-                    {data.metadata.pickup_location_pid}
-                  </Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>Transaction Location pid</Table.Cell>
-                  <Table.Cell width={12}>
-                    {data.metadata.transaction_location_pid}
-                  </Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>Transaction User pid</Table.Cell>
-                  <Table.Cell width={12}>
-                    {data.metadata.transaction_user_pid}
-                  </Table.Cell>
-                </Table.Row>
-                <Table.Row>
-                  <Table.Cell width={4}>State</Table.Cell>
-                  <Table.Cell width={12}>{data.metadata.state}</Table.Cell>
-                </Table.Row>
-              </Table.Body>
-            </Table>
-          </Grid.Column>
+        <Grid.Column>
```

Comment:
just as info: there is a presentation component which renders metadata fields (`MetadataTable` in `backoffice/components` if someone would like to reuse.

Reply:
@kprzerwa you are right I noticed it recently!